ArcGISRuntimeSDKQt_QMLSamples/Scenes/DisplayScenesInTabletopAR/ArcGISArView.qml

```diff
@@ -0,0 +1,234 @@
+/*******************************************************************************
```

Comment:
This file shouldn't be checked in as part of the sample. It should just reference the Toolkit and pull it in.